fedimint-server/src/config/api.rs

```diff
@@ -94,6 +94,17 @@ impl ConfigGenApi {
         Ok(state)
     }
 
+    fn require_status_maybe(
```

Comment:
nit: why call it `_maybe`? Isn't this more of a `require_any_status` fn?

Reply:
renamed